snap-packages/from-source/build.xml

```diff
@@ -0,0 +1,173 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+<project name="netbeans-snapcraft" basedir="." default="usage">
+    
+    <property name="build.dir" location="build"/>
+    <property name="dev.dir" location="${build.dir}/netbeans-dev"/>
+    <property name="rel.dir" location="${build.dir}/netbeans"/>
+    <property name="root.dir" location="../../.."/>
+    <tstamp>
+        <format property="build.date" pattern="yyyyMMdd"/>
+    </tstamp>        
+    
+    <available property="has.dev.snap" file="${dev.dir}/netbeans-dev_${build.date}_amd64.snap"/>
+    <available property="has.rel.snap" file="${rel.dir}/netbeans_${release.version}_amd64.snap"/>
+
+    <target name="usage">
+        <echo><![CDATA[
+        
+        Creating a Release:
+        
+          ant -Dreleaase.version=<version> -Drelease.branch=<branch> snap-rel
+     ]]></echo>
+    </target>
+    <target name="prepare-dev">
+
+        <mkdir dir="${dev.dir}/snap/gui"/>
+        <copy file="../frame512.png" todir="${dev.dir}/snap/gui"/>
```

Comment:
If it's of interest, I had to fix up the NetBeans SVG to get it to work in GNOME for the deb - https://github.com/apache/netbeans-tools/blob/master/nbpackage/src/main/resources/org/apache/netbeans/nbpackage/apache-netbeans.svg

Reply:
Might use that one as well later.